create_science_data/Create_example_level1_wfi_image.ipynb

```diff
@@ -0,0 +1,789 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2429af29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "random_data.meta.exposure.type"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "079159ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# take a look at the data\n",
+    "fig = plt.figure(figsize=(10, 5))\n",
+    "fig.add_subplot(1,3,1)\n",
+    "plt.imshow(random_data.data[0], origin=\"lower\")\n",
+    "fig.add_subplot(1,3,2)\n",
+    "plt.imshow(random_data.data[1], origin=\"lower\")\n",
+    "fig.add_subplot(1,3,3)\n",
+    "plt.imshow(random_data.data[2], origin=\"lower\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24dbbe9f",
+   "metadata": {},
+   "source": [
+    "### We can open the file we created from the fits data for comparison\n",
+    "\n",
+    "Obviously the NICMOS data isn't the same expected size as the WFI data that we expect. \n",
+    "We do expect to create level 1 files that are smaller, for example, the guide window arrays.\n",
+    "\n",
+    "The following is meant to show how to read a fits file and move the data arrays over to the roman datamodel for saving.\n",
+    "Validation on the datamodel and within the romancal code should take care of mismatches.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "156cefe9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_from_fits = roman_datamodels.open('level1_image_ramp_from_nicmos.asdf')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "569826b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_from_fits.data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "793399b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(10, 30))\n",
+    "fig.add_subplot(1,5,1)\n",
+    "plt.imshow(data_from_fits.data[0], origin=\"lower\")\n",
+    "fig.add_subplot(1,5,2)\n",
+    "plt.imshow(data_from_fits.data[1], origin=\"lower\")\n",
+    "fig.add_subplot(1,5,3)\n",
+    "plt.imshow(data_from_fits.data[2], origin=\"lower\")\n",
+    "fig.add_subplot(1,5,4)\n",
+    "plt.imshow(data_from_fits.data[3], origin=\"lower\")\n",
+    "fig.add_subplot(1,5,5)\n",
+    "plt.imshow(data_from_fits.data[4], origin=\"lower\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fa7d8e5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_from_fits.to_flat_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "434eb8f7",
+   "metadata": {},
+   "source": [
+    "## Open an example file with reasonably populated metadata"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "904d161b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_data=roman_datamodels.open('r0019103009008012017_09309_0012_WFI01_uncal.asdf')"
```

Comment:
For the example of reasonable meta data files, be sure to update the file name and upload the files to box.

Reply:
done